web/client/epics/widgetsbuilder.js

```diff
@@ -29,8 +30,11 @@ const getFTSelectedArgs = (state) => {
 };
 module.exports = {
     openWidgetEditor: action$ => action$.ofType(NEW, EDIT)
-        .switchMap(() => Rx.Observable.of(setControlProperty("widgetBuilder", "enabled", true))),
-    closeWidgetEditorOnFinish: action$ => action$.ofType(INSERT)
+        .switchMap(() => Rx.Observable.of(
+            setControlProperty("widgetBuilder", "enabled", true),
+            setControlProperty("metadataexplorer", "enabled", false)
```

Comment:
Should we disable also `annotations `and `get features info` panels?

Reply:
Should I use UPDATE_MAP_LAYOUT action and check if right > 0? 
Unfortunately I can not close annotations (also featuregrid do not do that).

Reply:
I think we should keep this behaviours for now because is not possible to have a complete management of annotations panel.